main.js

```diff
@@ -53,6 +53,10 @@ async function run() {
   const repository = core.getInput('repository', { required: true });
   const pagesBranch = core.getInput('pagesBranch', { required: true });
   const pagesFile = core.getInput('pagesFile', { required: true });
+  var ignorePrereleasesIn =  core.getInput('ignorePrereleases', { required: true });
```

Comment:
Could you change this to `getBooleanInput` as per https://github.com/actions/toolkit/tree/main/packages/core#inputsoutputs? This case seems handled in the core library.

Reply:
(and change it to const instead of var)